tests/Tests/100__deploy/100__installation/104__dashboard.robot

```diff
@@ -29,8 +29,8 @@ Verify Dashboard Is Shipped And Enabled Within ODS
     OpenShift Resource Field Value Should Be Equal As Strings    spec.ports[0].targetPort    8443    @{dashboard_services_info}
     OpenShift Resource Field Value Should Match Regexp    spec.clusterIP    ^((25[0-5]|(2[0-4]|1[0-9]|[1-9]|)[0-9])(\.(?!$)|$)){4}$
     ...    @{dashboard_services_info}
-    OpenShift Resource Field Value Should Be Equal As Strings    status.readyReplicas    2    @{dashboard_replicasets_info}
-    OpenShift Resource Field Value Should Be Equal As Strings    status.replicas    2    @{dashboard_replicasets_info}
+    OpenShift Resource Field Value Should Be Equal As Strings    status.readyReplicas    5    @{dashboard_replicasets_info}
```

Comment:
I think you should check if version is greater of equal to 1.17

Reply:
done